src/api/customer/customer-controller.ts

```diff
@@ -15,6 +15,35 @@ const createCustomer = (): CustomerCreationRequestHandler[] => {
     ]
 }
 
+const getAllCustomers = (): GetAllCustomerRequestHandler[] => {
+    return [
+        getAllCustomersValidator(),
+        paginationSerializer,
+        getAllCustomersBusiness,
+        getAllCustomersSerializer,
+        (req, res) => {
+            res.status(OK).json(res.locals.customersToRespond);
+        },
+    ];
+};
+
+const getCustomersById = (): GetCustomerByIdRequestHandler[] => {
+    return [
+        getCustomerValidator(),
+        getCustomerByIdBusiness,
+        getCustomerSerializer,
+        (req, res) => {
+            if (!res.locals.customerToRespond) {
+                res.status(NOT_FOUND).json(res.locals.customerToRespond);
+            } else {
+                res.status(OK).json(res.locals.customerToRespond);
+            }
```

Comment:
Deixar de fazer a negação e inverter a ordem do `if` é uma boa prática.

```suggestion
            if (res.locals.customerToRespond) {
                res.status(OK).json(res.locals.customerToRespond);
            } else {
                res.status(NOT_FOUND).json(res.locals.customerToRespond);
            }
```